README.md

```diff
@@ -92,15 +93,12 @@ py.test -v tests
 ## Releasing
 
 1. Increment `__version__` in `setup.py`
-1. Summarize release changes in `NEWS.md`
-1. Commit these changes, then tag the release
-  ```bash
+2. Summarize release changes in `NEWS.md`
+3. Commit these changes, then tag the release
+```bash
 git add setup.py NEWS.md
 git commit -m"Bump to version x.x.x"
 git tag -a -m"x.x.x" x.x.x
 git push --follow-tags
-  ```
-1. Build and release the new package
-  - `pip install wheel`
-  - `python setup.py sdist bdist_wheel`, then copy the `dist/*` to the pypiserver, OR
-  - `python setup.py sdist upload -r <server>` if you have that set up in your `.pypirc` file
+```
+4. Actions will build and release the package
```

Comment:
Can we specify "GH Actions", in case a dev who is not familiar with the context reads this?

Reply:
And it would probably be good to guide the reader to the actions config file.

Reply:
:+1: